assets/app/views/site/add.js

```diff
@@ -5,10 +5,10 @@ var _ = require('underscore');
 
```

Comment:
Why would these files need "View" in their name when they're in a "view" folder? It might be unnecessary.